taskcluster/ci/beetmover/kind.yml

```diff
@@ -24,6 +24,7 @@ only-for-build-types:
 job-template:
     attributes:
         artifact_map: taskcluster/fenix_taskgraph/manifests/fenix_candidates.yml
+        shipping_phase: ship
```

Comment:
In gecko the yml has "shipping-phase" with a hyphen, and that gets turned into a "shipping_phase" attribute by the "task" transform.  Can we align here somehow?

Reply:
Yeah, I first started using `shipping-phase` but it turns out it's only defined in `gecko_taskgraph` not in the standalone one. It's actually transformed by the `task` transform. @ahal Do you think we should port these bits of the tranform to standalone taskgraph?

[1] https://searchfox.org/mozilla-central/rev/0069a0cdc3cd136ea7411521219df6665f260be7/taskcluster/gecko_taskgraph/transforms/task.py#1958